pkg/cmd/experimental/syncgroups/grouplister.go

```diff
@@ -0,0 +1,103 @@
+package syncgroups
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/util"
+
+	osclient "github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/experimental/syncgroups/interfaces"
+	syncutil "github.com/openshift/origin/pkg/cmd/experimental/syncgroups/util"
+)
+
+// NewAllOpenShiftGroupLister returns a new AllLocalGroupLister
+func NewAllOpenShiftGroupLister(host string, groupClient osclient.GroupInterface) interfaces.LDAPGroupLister {
```

Comment:
Better name of `host`.  `ldapURL`?  Add to the godoc explaining how the `ldapURL` will be used.